src/Altinn.Platform/Altinn.Platform.Register/Register/Startup.cs

```diff
@@ -84,6 +87,11 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton(Configuration);
             services.Configure<GeneralSettings>(Configuration.GetSection("GeneralSettings"));
+            services.Configure<KeyVaultSettings>(Configuration.GetSection("kvSetting"));
+            services.Configure<AccessTokenSettings>(Configuration.GetSection("AccessTokenSettings"));
+
+            services.AddSingleton<IAuthorizationHandler, AccessTokenHandler>();
```

Comment:
Need to add ISigningKeyResolver as well.

Reply:
Fixed

Reply:
Solved